### PR TITLE
Add deletion of meter readings if submitting abstraction volumes to a…

### PR DESCRIPTION
**Pull request by DGajewska**
…pplyCleanUp

WATER-2063

Was receiving 500 error on submission of abstraction volumes if meter readings existed in data. Add deletion of meter readings if reading method is abstraction volumes.

### CODE REVIEW

File: src/modules/returns/lib/return-helpers.js [hunk 1]
```diff
@@ -466,6 +466,11 @@ const applyCleanup = (data, request) => {
         };
       });
     }
+    // Delete startReading and meter readings if submitting abstractionVolumes
+    if (get(updated, 'reading.method') === 'abstractionVolumes') {
+      delete updated.meters[0].startReading;
```
Comment (james-siteclick) [inline src/modules/returns/lib/return-helpers.js]: meters[0] may be undefined.
could consider:
```
updated.meters = updated.meters.map(meter => {
  delete meter.startReading;
  delete meter.readings;
})
```

File: test/modules/returns/lib/return-helpers.js [hunk 2]
```diff
@@ -891,6 +898,18 @@ experiment('applyCleanup', () => {
...
+      expect(result.meters[0].readings).to.not.exist();
+      expect(result.meters[0].startReading).to.not.exist();
+    });
+
+    test('does not remove meter readings and startReading if method is not abstractionVolumes', async () => {
+      const result = applyCleanup(createReturn(), createRequest());
+      expect(result.meters[0].readings).to.exist();
+      expect(result.meters[0].startReading).to.exist();
+    });
   });
```
Comment (james-siteclick) [inline test/modules/returns/lib/return-helpers.js]: Maybe add a test here for when meters is an empty array

**Comment by james-siteclick** [comment]
:shipit:
